src/trivial-shell.nix

```diff
@@ -1,10 +1,12 @@
-{ pkgs ? import <nixpkgs> {} }:
+{ pkgs ? import <nixpkgs> { } }:
 
-pkgs.mkShell {
+with pkgs;
```

Comment:
I know I'm not very involved with Lorri these days, but my opinion is thaht using `with` in such a large context is an antipattern. I'd be +1 in using for just the list in `buildInputs` though.